webhook-bridge/svix-webhook-bridge.example3.yaml

```diff
@@ -0,0 +1,74 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on when receiving webhooks
+receiver_http_addr: "0.0.0.0:5000"
+
+# Global svix settings.
+# A svix client is needed by each sender.
+# Each sender can override any of these fields by defining a `svix_client` section.
+global_svix_client:
+  token: "XXXXX"
+  options:
+    debug: true
+    server_url: http://example.com/
+
+# Webhook Senders configure an input and (optional) transformation.
+# Use transformations to conform the payload we get as input to the expected shape (app_id, message, etc).
+# When transformation is left unset, the payload is used as-is.
+senders:
+  - name: "rabbit-local"
+    input:
+      type: "rabbitmq"
+      queue: "local"
+      uri: "amqp://example.com/%2f"
+      requeue_on_nack: true
+    transformation: |
+      handler = (x) => ({ app_id: "app_1234", message: { eventType: "foo.bar", payload: x }})
+    # Can optionally specify`svix_client` here, but can be omitted entirely if the global is set.
+
+  - name: "rabbit-local2"
+    input:
+      type: "rabbitmq"
+      queue: "local2"
+      uri: "amqp://example.com/%2f"
+      requeue_on_nack: true
```

Comment:
I was reluctant to default the requeue since I have no idea which is more likely... however:

- SQS only lets you requeue (there's no choice)
- This is controlled by the way the topic is defined (outside of bridge entirely) for GCP Pub/Sub

We could default to `true` to fit with SQS, but it also feels weird to bias based on a single backend :shrug: